### PR TITLE
Zcx/llama2 infer 910b

### PR DESCRIPTION
**Pull request by zhaochaoxing**
## Motivation and Context

infer llama2 on 910b

## Description

For lightllm.

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Comment by yangbofun** [comment]
补一下测例？

**Comment by zhaochaoxing** [comment]
> 补一下测例？

补了。

**Review by jinminxi104 — APPROVED** [review]

File: impl/ascend_npu/diopi_impl/functions_ext/token_attention_inference.cpp [hunk 1]
```diff
@@ -38,4 +38,46 @@ diopiError_t diopiTokenAttentionInference(diopiContextHandle_t ctx, diopiTensorH
     END_CALL_ACL_OP();
 }
 
+diopiError_t diopiPagedAttention(diopiContextHandle_t ctx, diopiTensorHandle_t out, diopiConstTensorHandle_t q, diopiConstTensorHandle_t k,
+                                 diopiConstTensorHandle_t v, diopiSize_t actualSeqLengths, int64_t numHeads, int64_t numKeyValueHeads, int64_t dim,
+                                 diopiConstTensorHandle_t blockTable, int64_t blockSize) {
+    BEGIN_CALL_ACL_OP(out, q, k, v, blockTable);
+    at::IntArrayRef actSeqLen(actualSeqLengths.data, actualSeqLengths.len);
+    if (qAt.dim() == 2) {
```
Comment (jinminxi104) [inline impl/ascend_npu/diopi_impl/functions_ext/token_attention_inference.cpp:46]: 这边需要assert一下S只能是1

**Review by jingguo-st — APPROVED** [review]

File: impl/ascend_npu/diopi_impl/functions_ext/token_attention_inference.cpp (hunk 1 above)
Comment (zhaochaoxing) [reply impl/ascend_npu/diopi_impl/functions_ext/token_attention_inference.cpp:48]: 加上了

File: diopi_test/python/configs/diopi_configs.py [hunk 2]
```diff
@@ -8347,6 +8347,38 @@
         ),
     ),
 
+    'rotary_emb_v2': dict(
+        name=['rotary_emb_v2'],
+        interface=['CustomizedTest'],
+        dtype=[np.float32],
```
Comment (yangbofun) [inline diopi_test/python/configs/diopi_configs.py:8353]: 加一下float16？

File: diopi_test/python/configs/diopi_configs.py [hunk 3]
```diff
@@ -8347,6 +8347,38 @@
...
+        atol=1e-2,
```
Comment (yangbofun) [inline diopi_test/python/configs/diopi_configs.py:8354]: atol、rtol的设置对float32生效，atol_half，rtol_half对float16生效，这里，不写会有默认值，推荐使用默认(精度要求高)，如果默认不过，再适当调小atol和rtol。

File: diopi_test/python/configs/diopi_configs.py [hunk 4]
```diff
@@ -8593,6 +8625,144 @@
...
+            maxInputLen=[2,],
+            actualSeqLengths=[[2,2],],
+            numHeads=[32,],
+            numKeyValueHeads=[32,],
+            dim=[128,],
+        ),
+        tensor_para=dict(
+            args=[
+                {
+                    "ins": ['out'],
```
[the remark this answers is not in the document]
Comment (zhaochaoxing) [reply diopi_test/python/configs/diopi_configs.py]: done

File: diopi_test/python/conformance/global_op_list.py [hunk 5]
```diff
@@ -71,6 +71,10 @@
                    "clip_grad_norm_": {"tensors"},
                    "apply_penalty": {"logits"},
                    "context_attention": {"out"},
+                   "prompt_flash_attention": {"out"},
```
[the remark this answers is not in the document]
Comment (zhaochaoxing) [reply diopi_test/python/conformance/global_op_list.py]: done

File: diopi_test/python/configs/diopi_configs.py (hunk 2 above)
Comment (zhaochaoxing) [reply diopi_test/python/configs/diopi_configs.py]: done

File: diopi_test/python/configs/diopi_configs.py (hunk 3 above)
Comment (zhaochaoxing) [reply diopi_test/python/configs/diopi_configs.py]: done

**Review by yangbofun — APPROVED** [review]

**Pull request merged**